pybossa/syncer/project_syncer.py

```diff
@@ -85,6 +85,10 @@ def _build_payload(self, project, target=None):
             payload = self._merge_pass_through_keys(
                     project_dict, target)
 
+        input_data = project['info'].get('data_classification', {}).get('input_data') or ''
```

Comment:
1. This chunk of code should be under `if not target:` as the original issue happens only when creating project via sync.
2. Can consider using `payload['info'].get('annotation_config')` instead of payload['info']['annotation_config'] to avoid keyError

Reply:
It looks to me we don't need this chunk of code. Because "if not target", it is a new project. It will enter [here](https://github.com/bloomberg/pybossa/blob/e88ffd26869310b2ac3812fda57e7731cd933318/pybossa/syncer/project_syncer.py#L49). And "_create" calls a project API "post" method, which enters this [line](https://github.com/bloomberg/pybossa/blob/master/pybossa/api/project.py#L58).

Reply:
We discussed yesterday that we don't want to hit `set_default_amp_store(data)` function when syncing to a new L1/L2 project, that's why we need to explicitly set `amp_store=False` before we send POST API.